fluent-dom/.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
To support async/await, but I think I'll enable it in the main .eslintrc instead